src/js/lotto.js

```diff
@@ -36,13 +48,50 @@ class Lotto {
     this.#state.lottos = games.map(this.#generateLottoNumbers);
   }
 
+  checkWinnerNumber($winningNumbers, $bonusNumber) {
+    const winningNumbers = getInputNumberValues($winningNumbers);
+    const bonusNumber = Number($bonusNumber.value);
+
+    const numberOfMatchedNumbers = this.#state.lottos.map((lotto) => {
+      const matchedValueCount = getMatchedValueCountInArray(
+        lotto,
+        winningNumbers
+      );
+      const hasBonusNumber = !!lotto.find((num) => num === bonusNumber);
```

Comment:
```suggestion
      const hasBonusNumber = lotto.find((num) => num === bonusNumber) != null;
```
이렇게 작성하는게 더 선언적인 것 같아요.

find의 반환값이 number일수도 undefined일수도 있는데 형태가 다른 두 타입을 not을 통해 암시적으로 다루는 형태는 위에 작성된 형태보다 이해하기 어려울 수도 있겠다고 생각이 들어요 👀